### PR TITLE
bluetooth: tester: Add MESH and LE Audio overlays

### PR DESCRIPTION
**Pull request by sjanc**
Initially I was hoping we can avoid dedicated overlays and have all config options enabled but it turned out that built-in services may affect other tests (especially core related like GAP and GATT).

### CODE REVIEW

**Comment by sjanc** [comment]
@mkasenberg @KKopyscinski   FYI   this will require update on autopts bot once merged

File: tests/bluetooth/tester/overlay-le-audio.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Volume Offset Control Service
+CONFIG_BT_VOCS_MAX_INSTANCE_COUNT=2
+CONFIG_BT_VOCS_CLIENT_MAX_INSTANCE_COUNT=2
+
+# Audio Input Control Service
+CONFIG_BT_AICS_MAX_INSTANCE_COUNT=4
+CONFIG_BT_AICS_CLIENT_MAX_INSTANCE_COUNT=4
+
+#Volume Control
```
Comment (Thalley) [inline tests/bluetooth/tester/overlay-le-audio.conf:17]:
```suggestion
# Volume Control
```

File: tests/bluetooth/tester/overlay-le-audio.conf [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# LE Audio Section
```
Comment (Thalley) [inline tests/bluetooth/tester/overlay-le-audio.conf:1]: I think the file name already gives this away ;)

**Review by Thalley — DISMISSED** [review]
Minor comments, but nothing that needs to be fixed

**Review by Thalley — APPROVED** [review]

**Review by MariuszSkamra — APPROVED** [review]

**Pull request merged**